<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchReq</class>
 <widget class="QDialog" name="SearchReq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Your cell:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>â„–:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_cell">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Calories:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_cal">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Fat:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fat">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Carb:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_carb">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Fiber:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fib">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Protein:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_pro">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_type">
          <property name="text">
           <string>NoN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7"/>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>90</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
